docs/docs/predict-tutorial.md

```diff
@@ -0,0 +1,327 @@
+        "n_frames": 16,
+    },
+    total_frames=16,
+)
+```
+
+The full recommended `VideoLoaderConfig` for the `slowfast` model is:
+```python
+video_loader_config = VideoLoaderConfig(
+    video_height=224,
+    video_width=224,
+    crop_bottom_pixels=50,
+    ensure_total_frames=True,
+    megadetector_list_config={
+        "confidence": 0.25,
+        "fill_mode": "score_sorted",
+        "n_frames": 32,
+    },
+    total_frames=32,
+)
+```
+
+You can see the full default configuration for each model in `models/config`<!-- TODO: add link to source and update if needed><!-->. For detailed explanations of all possible configuration arguments, see [All Optional Arguments](configurations.md).
+
+## Default behavior
+
+In each case, `zamba` will output a `.csv` file with rows labeled by each video filename and columns for each class (ie. species). The default prediction will store all class probabilities, so that cell (i,j) can be interpreted as *the probability that animal j is present in video i.* 
+
+By default, predictions will be saved to `zamba_predictions.csv`. You can save predictions to a custom path using the `--save-path` argument.
+
+```console
+$ cat zamba_predictions.csv
+filepath,aardvark,antelope_duiker,badger,bat,bird,blank,cattle,cheetah,chimpanzee_bonobo,civet_genet,elephant,equid,forest_buffalo,fox,giraffe,gorilla,hare_rabbit,hippopotamus,hog,human,hyena,large_flightless_bird,leopard,lion,mongoose,monkey_prosimian,pangolin,porcupine,reptile,rodent,small_cat,wild_dog_jackal
+example_vids/eleph.MP4,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,1.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
+example_vids/leopard.MP4,0.0,0.0,0.0,0.0,2e-05,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0125,0.0,0.0,0.0,0.0,0.0,0.0,0.0,1.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
+example_vids/blank.MP4,0.0,0.0,0.0,0.0,0.0,1.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
+example_vids/chimp.MP4,0.0,0.0,0.0,0.0,0.0,0.0,1e-05,0.0,1.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,1e-05,4e-05,0.00162,0.0,0.0,0.0,0.0,0.0,2e-05,2e-05,0.0,1e-05,0.0,0.0038,4e-05,0.0
+```
+
+## Step-by-step tutorial
+
+### 1. Specify the path to your videos
+
+Save all of your videos in one folder.
+
+* Your videos should all be saved in formats that are suppored by FFMPEG, [which are listed here](https://www.ffmpeg.org/general.html#Supported-File-Formats_002c-Codecs-or-Features).
+* Your video folder must contain only valid video files, since zamba will try to load all of the files in the directory.
+* Your videos must all be in the top level of the video folder - `zamba` does not extract videos from nested directories.
+
+Add the path to your video folder with `--data-dir`. For example, if your videos are in a folder called `example_vids`:
+
+```console
+$ zamba predict --data-dir example_vids/
+```
+
+In the python package, the data directory is specified when `PredictConfig` is instantiated:
```

Comment:
I agree, it's a tough call. I think the downside of a separate python page is potentially repeating information from the tutorials. 

A couple options:

- We could move all of the python references / code from the tutorial pages, and have a separate python page. The python page could have basic usage and all of the code snippets that are currently in the tutorials, but minimal explanation - instead of repeating information we point people to the relevant other pages. 
- To me, the most cumbersome python in the tutorials is the basic usage at the top. We could move the basic python usage sections from the predict and train tutorial to a 'basic python usage' page, but keep the python snippets in the step-by-step tutorials. I think this is my preference.

@ejm714 @pjbull any strong preferences here?